4-WebApp-your-API/4-3-AnyOrg/TodoListService/appsettings.json

```diff
@@ -5,7 +5,9 @@
     "TenantId": "common",
     "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
     "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
-    "GraphAPI": "https://graph.microsoft.com/v1.0"
+    "GraphAPI": "https://graph.microsoft.com/v1.0",
+    "ClientCapabilities": [ "cp1" ],
+    "AllowedTenants": [ "here are your allowed tenants" ]
```

Comment:
>here are your allowed tenants [](http://example.com/codeflow?start=25&length=29)

add the tenant ids of the tenants from where you want the client applications to call your Web API